src/components/Header.js

```diff
@@ -2,25 +2,108 @@ import React from 'react';
 
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
-// import IconButton from '@material-ui/core/IconButton';
-// import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+
+
+
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import Typography from '@material-ui/core/Typography';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+
+import './Header.scss';
 
 import PropTypes from 'prop-types';
 
-const Header = (props) =>
-    <AppBar position="static" className="header">
-      <Toolbar>
-        {/*<Link to="/" style={{ textDecoration: 'none', display: 'block' }}>*/}
-          {/*<IconButton color="inherit" aria-label="Home" >*/}
-            {/*<ChevronLeftIcon />*/}
-          {/*</IconButton>*/}
-        {/*</Link>*/}
-        {props.headline}
-      </Toolbar>
-    </AppBar>;
-
-Header.proppTypes = {
-  headline: PropTypes.string.isRequired
+class Header extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      anchorEl: null
+    };
+  }
+
+  handleMenu = (event) => {
+    if (!this.props.user) {
+      this.props.handleClickLoginLogout();
+    } else {
+      this.setState({ anchorEl: event.currentTarget });
+    }
+  };
+
+  handleClose = () => {
+    this.setState({ anchorEl: null });
+  };
+
+  render() {
+    const open = Boolean(this.state.anchorEl);
+
+    return <div className="geovation-header">
+      <AppBar position="static" className="header">
+        <Toolbar>
+          <IconButton color="inherit" aria-label="Menu" className="menuButton">
+            <MenuIcon />
+          </IconButton>
+
+          <Typography variant="h6" color="inherit" className="grow">
+            {this.props.headline}
+          </Typography>
+
+          <div>
+            <IconButton
+              // aria-owns={open ? 'menu-appbar' : undefined}
```

Comment:
why comment this out?

Reply:
you will use it if need to open the menu